server/models/Request.js

```diff
@@ -19,6 +19,7 @@ const requestSchema = new Schema({
     required,
     validate: {
       validator: function (start) {
+        if (!this.isNew) return true;
```

Comment:
I don't see a field called `isNew` here. Is this intentional? Can you elaborate if so?

Reply:
`isNew` is an inherited property for all mongoose schemas. It will be true on create, but false on update. Upon further thought, it looks like this line should be omitted

Reply:
Ah gotcha!